tests/unit/EnvTest.php

```diff
@@ -225,14 +268,224 @@ public function testGet_trueTitlecase()
     public function testGet_trueUppercase()
     {
         // Arrange
-        $name = 'TEST_TRUE_UPPERCASE';
-        $this->putEnv($name . '=TRUE');
+        $varname = 'TESTGET_TRUEUPPERCASE';
+        $this->putEnv($varname . '=TRUE');
         $expected = true;
         
         // Act
-        $actual = Env::get($name);
+        $actual = Env::get($varname);
+
+        // Assert
+        $this->assertSame($expected, $actual);
+    }
+    
+    public function testRequireEnv_exists()
+    {
+        // Arrange
+        $varname = 'TESTREQUIREENV_EXISTS';
+        $this->putEnv($varname . '=exists');
+        $expected = 'exists';
+        
+        // Actual
+        $actual = Env::requireEnv($varname);
+
+        // Assert
+        $this->assertSame($expected, $actual);
+    }
+    
+    public function testRequireEnv_notfound()
+    {
+        // Arrange
+        $varname = 'TESTREQUIREENV_NOTFOUND';
+        
+        // Actual
+        $this->expectException(EnvVarNotFoundException::class);
+        Env::requireEnv($varname);
+
+        // Assert
+        $this->fail("Should have thrown EnvVarNotFoundException.");
+    }
+    
+    public function testRequireEnv_empty()
+    {
+        // Arrange
+        $varname = 'TESTREQUIREENV_EMPTY';
+        $this->putEnv($varname . '=');
+        
+        // Actual
+        $this->expectException(EnvVarNotFoundException::class);
+
+        Env::requireEnv($varname);
+
+        // Assert
+        $this->fail("Should have thrown EnvVarNotFoundException.");
+    }
+   
+    public function testGetArray_notFoundExists()
+    {
+        // Arrange
+        $varname = 'TESTGETARRAY_NOTFOUNDEXISTS';
```

Comment:
I'm confused by this test/var name. Is this supposed to just be `..._NOTFOUND` (rather than `..._NOTFOUNDEXISTS`), since it won't exist?